### PR TITLE
J. K. Letters, Ch. 1-2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal-letters/modernized/04-chapter-1.adoc [hunk 1]
```diff
@@ -926,8 +1002,8 @@ with desires that you may witness a renewal of strength
 to persevere in the path of peace,
 by looking to the Lord for help, who is the never-failing helper of his people.
 
+[.signed-section-closing]
 I remain in near affection, your loving cousin,
 
+[.signed-section-signature]
 J+++.+++ K.
-
-Colchester, 3rd 10th Month, 1763.
```
Comment (jaredh159) [inline journal-letters/modernized/04-chapter-1.adoc:933]: Did this line get accidentally deleted?

File: journal-letters/original/04-chapter-1.adoc [hunk 2]
```diff
@@ -927,6 +997,5 @@ by looking to the Lord for help, who is the never-failing helper of his people.
 
 I remain in near affection, thy loving cousin,
 
+[.signed-section-signature]
 J+++.+++ K.
-
-Colchester, 3rd 10th Month, 1763.
```
Comment (jaredh159) [inline journal-letters/original/04-chapter-1.adoc:932]: same here...

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Looks really good.  Just one thing that caught my eye (unless I’m misreading the diffs...)

Everything else is great.  I'm glad you did not embed all of these letters, that was the right call, since this is pretty much 100% letters. 👍

File: journal-letters/modernized/04-chapter-1.adoc (hunk 1 above)
Comment (maijapapaya) [reply journal-letters/modernized/04-chapter-1.adoc:933]: It actually belonged to the next letter that begins on chapter two so I moved it over to that.

Comment (jaredh159) [reply journal-letters/modernized/04-chapter-1.adoc:933]: ahh, that makes sense. thanks, merging now. 👍

**Review by jaredh159 — APPROVED** [review]

**Pull request merged**